README.md

```diff
@@ -135,7 +135,7 @@ Aside from basic statistics like free memory, disk, load averages, etc, we have
 
 ## delayed_job
 
-Install monitoring and automatic startup for delayed job workers via monit. You MUST set the job queues as follows:
+Install monitoring and automatic startup for delayed job workers via monit. You MUST set the `job_queues` variable as follows:
```

Comment:
Add an example of how to get multiple queus, it's not at all clear how to say I want 4 export workers